packages/compass-components/src/components/spin-loader.tsx

```diff
@@ -37,4 +49,6 @@ function SpinLoader({ size = '12px', title }: SpinLoaderProps): JSX.Element {
   );
 }
 
+const SpinLoader = withTheme(UnstyledSpinLoader) as typeof UnstyledSpinLoader;
```

Comment:
Until that `withTheme` hook is cleaned up a bit, I'm thinking it's probably cleaner to use the `useDarkMode` hook inside of the function.

Reply:
Aah I already forgot we had that

Reply:
I did some changes to the spinner in https://github.com/mongodb-js/compass/pull/3708/files#diff-74dca60361c92e5934a3dffeeca1949b3a6e223cdb72e261c4017022af5738f0 for the shell update, but this is blocked now, just FYI if you want to pick them up too